README.md

```diff
@@ -91,6 +92,12 @@ public class MainApplication extends Application implements ReactApplication {
 }
 ```
 
+### Windows Support
+
+Because this RNW package targets React Native 0.63.0+, you probably won't need to link it manually. Otherwise if it's not the case, follow these additional instructions. You also need to manual link the module on Windows when using React Native Windows prior to 0.63:
```

Comment:
```suggestion
Because this RNW package targets React Native 0.63.0+, you probably won't need to link it manually. Otherwise if it's not the case, follow these additional instructions. You also need to manually link the module on Windows when using React Native Windows prior to 0.63:
```